api/config.js

```diff
@@ -0,0 +1,63 @@
+import {createRemoteJWKSet, jwtVerify} from "jose";
+import request from "needle";
+
+const {AUTH_SERVER_URL, AUTH_REALM, GATEWAY_ACCOUNT_URL, GATEWAY_KEY} = process.env;
```

Comment:
This file implements a serverless lambda which exchanges an Identity Provider token for a Gateway token. It requires the environment variables above, as listed in the PR description.